Description: Let port monitoring (netstat -ant) no more cut IPv6 addresses
 /usr/lib/hobbit/client/bin/hobbitclient-linux.sh calls netstat -ant to
 get a list of all ports of the server.
 The default behavior of netstat -ant is to use fixed column width for
 addresses, which means that long IPv6 addresses are cut.
Author: Roland Rosenfeld <roland@spinnaker.de>
Bug-Debian: https://bugs.debian.org/734867
Reviewed-By: Axel Beckert <abe@debian.org>
Forwarded: http://lists.xymon.com/pipermail/xymon/2015-September/042224.html
Last-Update: 2015-09-10

Index: xymon/client/xymonclient-linux.sh
===================================================================
--- xymon.orig/client/xymonclient-linux.sh	2015-02-04 00:18:44.607857512 +0100
+++ xymon/client/xymonclient-linux.sh	2015-02-04 00:18:44.603857447 +0100
@@ -73,7 +73,7 @@
 netstat -s
 echo "[ports]"
 # Bug in RedHat's netstat spews annoying error messages. 
-netstat -antu 2>/dev/null
+netstat -antuW 2>/dev/null
 echo "[ifstat]"
 /sbin/ifconfig 2>/dev/null
 # Report mdstat data if it exists
